docs/content/2.api/2.options.md

```diff
@@ -60,3 +60,25 @@ const strapi = new Strapi({
 - Default: `{}`
 
 Options to be passed to the [Axios instance](https://github.com/axios/axios#request-config).
+
+### `axiosCreate`
+
+- Type: `AxiosStatic['create']`
+- Default: `(config) => axios.create(config)`
+
+Axios instance creator where the `axiosOptions` and merged internal defaults are passed as the first parameter. This is helpful if you want to customise the axios instance. 
+
+For example, lets say we want to implement request caching using the [`axios-cache-interceptor`](https://axios-cache-interceptor.js.org/) package:
+
+```ts
+import axios from "axios";
+import Strapi from "strapi-sdk-js"
+import { setupCache } from "axios-cache-interceptor"
+
+const strapi = new Strapi({
+  axiosCreate(config) {
+    const instance = axios.create(config)
+    return setupCache(instance)
+  }
```

Comment:
I don't think we should provide a way to override the creation of the instance as we can't perfectly control what will be returned and compatible with the SDK configuration. You can still do as below:
```ts
const strapi = new Strapi({
  // ...
})

strapi.axios = setupCache(strapi.axios)
```